### PR TITLE
chore/bump 0.3.0

### PR DESCRIPTION
**Pull request by 0xArdi**

### CODE REVIEW

File: packages/elcollectooorr/contracts/token_vault/tests/test_contract.py [hunk 1]
```diff
@@ -38,8 +38,6 @@
     TokenVaultFactoryContract,
 )
 
-from tests.conftest import ROOT_DIR
-
 
 DEFAULT_GAS = 1000000000
```
Comment (DavidMinarsch) [inline packages/elcollectooorr/contracts/token_vault/tests/test_contract.py:42]: There is a problem with this test file, it depends on `basket_factory` and `basket`. These integration tests need to move up to the agent level, as we don't want to change the dependencies unnecessarily just for the tests.

**Review by DavidMinarsch — COMMENTED** [review]
Looking good - just need to move some of the integration style tests to the agent level

File: .pylintrc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [MASTER]
 ignore-patterns=.*_pb2.py
-ignore-paths=^packages/valory/.*$
+ignore-paths=^packages/valory/.*$,.*/tests/.*
```
[the remark this answers is not in the document]
Comment (0xArdi) [reply .pylintrc:3]: too many linting errors that were not previously ignored were present

[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply .pylintrc:3]: I see - suggest doing a disable on the files, easier to spot where we need to later rmeove

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**